ansible/playbooks/roles/opensearch/tasks/install-ops-plugins.yml

```diff
@@ -0,0 +1,25 @@
+---
+# NOTE: Keep in mind compatibility matrix for Open Distro https://opendistro.github.io/for-elasticsearch-docs/docs/install/plugins/#plugin-compatibility
+- name: Install opensearch-* packages
+  package:
+    name: "{{ _packages[ansible_os_family] }}"
+    state: present
+  vars:
+    _packages:
```

Comment:
I think these variables should be moved to defaults/main.yml

Reply:
LGTM however due to embedded variables problem I will postpone it to separate feature/improvement.